src/api/const.ts

```diff
@@ -1,6 +1,6 @@
 import { getRefreshToken, getToken } from 'services/token.service'
 
-export const HOST = process.env.HOST
+export const HOST = process.env.HOST || 'http://localhost:5000'
```

Comment:
I don't think that you should add default value for host if it wasn't provided by enviroment

Reply:
Was created const <b>SOCKET_HOST_URL</b> which equal <b>process.env.SOCKET_HOST || DEFAULT_HOST_URL ('http://localhost:5000')</b>

Reply:
And HOST was chenged on <b>HOST_URL</b> than equals <b>process.env.HOST || DEFAULT_HOST_URL ('http://localhost:5000/')</b>